thrift-logger/src/main/java/com/pinterest/singer/client/logback/AuditableLogbackThriftLoggerFactory.java

```diff
@@ -36,32 +51,149 @@
 
```

Comment:
shall we have`AuditableLogbackThriftLoggerFactory` extends `LogbackThriftLoggerFactory`?

Reply:
refactored in the update of the PR.